Sources/Sentry/include/SentryTracer.h

```diff
@@ -20,10 +20,7 @@ static NSTimeInterval const SentryTracerDefaultTimeout = 3.0;
 
 @interface SentryTracer : NSObject <SentrySpan>
 
-/**
- *Span name.
- */
-@property (nonatomic, copy) NSString *name;
```

Comment:
l: instead of complete remove this, what do you think of having a readonly property that reads straight from the context? This way we dont need to change `tracer.name` to `tracer.transactionContext.name` everywhere, and keep future usage easier.

Reply:
But I already changed it "everywhere" 😅
Personally I think having one place to get something is better than having multiple roads. Let me know what you think.

Reply:
Well, I dont think we gonna use the name much more.
Since you already changed everything you can ignore this 😉